src/main/java/com/powsybl/openloadflow/network/impl/LfDanglingLineGenerator.java

```diff
@@ -71,7 +71,7 @@ public OptionalDouble getRemoteControlReactiveKey() {
 
     @Override
     public double getTargetQ() {
-        return getDanglingLine().getGeneration().getTargetQ() / PerUnit.SB;
+        return Networks.fixValue(getDanglingLine().getGeneration().getTargetQ()) / PerUnit.SB;
```

Comment:
I find the name fixValue unclear. Maybe redefineIfNan, or something similar?

Reply:
Done

Reply:
thanks!